virtualmouse/fakeinput.go

```diff
@@ -1,11 +1,39 @@
 package main
 
 import (
+	"fmt"
 	"github.com/BurntSushi/xgb"
 	"github.com/BurntSushi/xgb/xproto"
 	"github.com/BurntSushi/xgb/xtest"
 )
 
-func fakeinput (X *xgb.Conn, wid xproto.Window, x int16, y int16) {
-	xtest.FakeInput(X, 6, 0, 0, wid, x, y, 0)
+type connInfo struct {
```

Comment:
connInfo should start with a capital letter, or any package outside of this one won't be able to see it.
Basically starting with lowercase is equivalent to "private" in c++